src/lib/utils/paths.ts

```diff
@@ -0,0 +1,28 @@
+import * as Path from 'path';
+import { isArray } from 'util';
+import { Minimatch, IMinimatch } from 'minimatch';
+
+function convertToMinimatch(pattern: string): IMinimatch {
+    // Ensure uniform absolute path cross OS
+    // (POSIX would resolve c:/path to /path/to/repo/c:/path without this check)
+    if (Path.isAbsolute(pattern) && Path.sep === '/') {
+        pattern = pattern.replace(/^\w:/, '');
```

Comment:
Well i corrected this as it was a bit confusing (even for me). But as written in my tests this should now be correctly resolved according to the current project (with or without drive notation). I generally leverage the functionality  in `Path.resolve` which on Windows will resolve `/some/path`, by adding the drive letter of your current repo (eg. with a repo at `D:/my-repo/some/path` `Path.resolve('/some/path')` would return `D:/some/path`). There is no active validation whether the path actually exists or not here, but just a correction of the possible path.

However the drive letter removal is only necessary on systems using POSIX path system (everybody else but windows), hence the `Path.sep === '/'` check, as `C:/some/path` would otherwise resolve to `/myrepo/C:/some/path`

(you can try it for yourself by resolving paths using `Path.posix.resolve` or `Path.win32.resolve`)

Reply:
Just an update on the UNIX pathing system. I asked around and for UNIX systems (Linxus + MAC) you change drive by typing something like `/volumes/drive`. So absolute paths always start with a '/' on those systems. So it should be good like this.